packages/components/src/TooltipTrigger/TooltipTrigger.component.js

```diff
@@ -75,8 +93,9 @@ class TooltipTrigger extends React.Component {
 
 TooltipTrigger.propTypes = {
 	children: PropTypes.element,
-	label: PropTypes.string,
+	label: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
 	tooltipPlacement: OverlayTrigger.propTypes.placement,
+	ariaLabel: PropTypes.string,
```

Comment:
Please remove that.
Tooptip is something that appears, containing precisions you want to display. It's something that goes on a main element containing an info. It is only for people that can see.
For the others, you need the screen readers to read the precisions when user want to read the main element. If you need an aria label, it's on the main element you need to set it. Setting it on a tooltip has no sense.